tests/main/login/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Checks for snap login
+execute: |
+  echo "Checking missing email error"
+  expect -f missing_email_error.exp
+
+  echo "Checking wrong password error"
+  expect -f unsuccessful_login.exp
+
+  if [[ $SPREAD_STORE_USER && $SPREAD_STORE_PASSWORD ]]; then
+    echo "Checking successful login"
```

Comment:
We're using four spaces now for indenting the yaml and the shell inside it, so 8 spaces at the start here.